Source/options.h

```diff
@@ -25,6 +25,15 @@ struct AudioOptions {
 	bool bWalkingSound;
 	/** @brief Automatically equipping items on pickup emits the equipment sound. */
 	bool bAutoEquipSound;
+
+	/** @brief Output sample rate (Hz) */
+	Uint32 nSampleRate;
+	/** @brief The number of output channels (1 or 2) */
+	Uint8 nChannels;
+	/** @brief Buffer size (number of frames per channel) */
+	Uint32 nBufferSize;
+	/** @brief Quality of the resampler, from 0 (lowest) to 10 (highest) */
+	Uint8 nResamplingQuality;

```

Comment:
Ajenbo is currently busy replacing these with uint32_t and uint8_t.

Reply:
Well only in areas where it's not directly interacting with SDL, I think it should be correct at this point. At least to my understanding.